README.md

```diff
@@ -1,3 +1,4 @@
 # Simple-Photo-Page
 Photo Page made with HTML and CSS
-<img src="https://github.com/Rahul-k25/Simple-Photo-Page/blob/main/Final-result.PNG" width="230">
\ No newline at end of file
+<br>
+<img src="https://github.com/Rahul-k25/Simple-Photo-Page/blob/main/Final-result.PNG" width="100%">
```

Comment:
plz remove the inline style `width=100%`
All properties should be under the class selector